mangaki/mangaki/models.py

```diff
@@ -141,6 +141,7 @@ def retrieve_poster(self, url=None, session=None):
             return False
 
         filename = os.path.basename(urlparse(url).path)
+        # Hé mais ça va pas écraser des posters / créer des collisions, ça ?
```

Comment:
Oui mais c'est l'objet d'une issue nice-to-have (vais créer). Et tu es pédant avec le paradoxe des anniversaires.